pkg/microservice/aslan/core/common/service/environment.go

```diff
@@ -256,18 +257,46 @@ func ListWorkloadsInEnv(envName, productName, filter string, perPage, page int,
 	if err != nil {
 		return count, resp, err
 	}
-	fillServiceDisplayName(resp, productInfo)
+
+	fillServiceInfo(resp, productInfo)
 	return count, resp, nil
 }
 
 type FilterFunc func(services []*Workload) []*Workload
 
 type workloadFilter struct {
-	Name string `json:"name"`
+	Name            string      `json:"name"`
+	ServiceName     string      `json:"serviceName"`
```

Comment:
Use `service_name` instead of `serviceName` in json tag?

Reply:
this will involve client, i suggest refactor json tag style in one-time modification, which mainly involve environment related apis